charts/shoot-core/components/charts/vpn-shoot/templates/_helpers.tpl

```diff
@@ -1,9 +1,10 @@
+#doubt
 {{- define "vpn-shoot.certs.data" -}}
 ca.crt: {{ .Values.vpnShootSecretData.ca }}
 tls.crt: {{ .Values.vpnShootSecretData.tlsCrt }}
 tls.key: {{ .Values.vpnShootSecretData.tlsKey }}
 {{- end -}}
```

Comment:
how to incorporate this? 
`vpnShootSecretData:
  ca: ZHVtbXktYjY0LWRhdGE=
  tlsCrt: ZHVtbXktYjY0LWRhdGE=
  tlsKey: ZHVtbXktYjY0LWRhdGE=`
can this part be included in the code? 
If not, how to use it while creating the Secret object.

Reply:
You should probably define something like https://github.com/shafeeqes/gardener/blob/refactor/vpn-shoot/pkg/operation/botanist/component/metricsserver/metrics_server.go#L436-L441 and do it similar as in the `metricsserver` component. We can also do this together in a call next week.